report_xlsx/models/res_currency.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ResCurrency(models.Model):
+    _inherit = "res.currency"
+
+    def _get_format_amount(self):
+        """Get the format to be used in cells (symbol included).
+        Used in account_financial_report addon"""
+        return "%s#,##%s%s" % (
+            self.symbol if self.position == "before" else "",
```

Comment:
It isn't correct. This will render a string like
```python
'$ #,##0,00'
```
The symbol string should go between apostrophes:
```python
'"$" #,##0,00'
```

Easy way:
```python
f'"{self.symbol}"'
```